packages/accessibility/src/components/LiveRegion/LiveRegion.tsx

```diff
@@ -3,29 +3,30 @@ import React from 'react';
 export interface Props {
   id: string;
   announcement: string;
+  ariaLiveType: "polite" | "assertive" | "off";
 }
 
-// Hide element visually but keep it readable by screen readers
-const visuallyHidden: React.CSSProperties = {
-  position: 'fixed',
-  width: 1,
-  height: 1,
-  margin: -1,
-  border: 0,
-  padding: 0,
-  overflow: 'hidden',
-  clip: 'rect(0 0 0 0)',
-  clipPath: 'inset(100%)',
-  whiteSpace: 'nowrap',
-};
-
-export function LiveRegion({id, announcement}: Props) {
+export function LiveRegion({id, announcement, ariaLiveType = "polite"}: Props) {
```

Comment:
@ayy-bc Maybe we should keep this defaulted to `assertive` so it doesnt affect any existing usages

Reply:
I think assertive would be the way to go! -- User research indicated that this woul